org.openhab.binding.zigbee/README.md

```diff
@@ -198,6 +198,28 @@ If the usb dongle is not recognized, it might be necessary to make the dongle's
   ```
 - Plug in the dongle. It should now be recognized properly as ttyUSBx.
 
+##### Flashing/Upgrading on Linux
+
+Some Ember EZSP NCP coordinators can be flashed to upgrade the firmware using walthowd/husbzb-firmware (https://github.com/walthowd/husbzb-firmware).
+It has been tested with the HUSZBZ-1 and Telegesis ETRX357USB and may work with other coordinators based on similar EM358x/EM3581, ETRX35x/ETRX357, and EFR32MGxx MCU chips from Silicon Labs.
```

Comment:
Are you REALLY sure about this???

The EFR32 and EM357/358 are COMPLETELY different. Their firmware is not compatible at all so I'm surprised that this states that it updates all types of Silabs devices. I understood from the discussion on the forum that this was for the HUSBZB1 which is an EM357 - if so it is not going to work for other devices.

Reply:
I'm just repeating what was stated on the tool's github readme. That's actually a direct copy and paste from the tool. It's no biggie to me to remove it. I figured the developer of the tool knew what they were talking about.

Reply:
That sounds dodgy. With the EM357 and EM358 it's reasonably safe, but the EFR32 devices have a lot more configuration options, and if you program the wrong code (ie code that is configured to use a different oscillator, or different memory, or different ports) then you will brick the device. I've recently done exactly this when a customer changed their hardware configuration in an EFR32 and you then need to have the programmer to reprogram the device at low level.

I just don't want to be responsible for bricking peoples devices so maybe we should add a "use at own risk" disclaimer to this.

The other option of course is always to use the binding, although the user still needs to have the correct firmware in order to flash the device.

Reply:
Always use the binding only works if the binding supports it. My understanding is that's not the case for the HUSZBZ-1.

How about I end the sentence at "may work with other coordinators. Use at your own risk."

Reply:
> Always use the binding only works if the binding supports it. My understanding is that's not the case for the HUSZBZ-1.

Yes, that was also my understanding, but as I don't have one of these devices I've never been able to test it. However clearly this has a bootloader right, or it would be impossible to bootload using this other software. And presumably it is using the standard Silabs bootloader as it claims to be compatible with every Silabs chip. In this case, why wouldn't it also work with the binding, or the OH console at least? I added bootloader support originally for Eclipse SmartHome when German Telecom were directly using the binding and they needed it to support their customers.

> How about I end the sentence at "may work with other coordinators. Use at your own risk."

I would state something like "You are responsible to ensure you load the correct firmware for your device. Loading the wrong firmware may render your device inoperable". This makes it a bit clearer (IMHO) that the consequences of getting this wrong are a bricked stick that most people will not be able to recover from.